cypress/integration/home-page.spec.js

```diff
@@ -452,4 +452,132 @@ describe('Home page', () => {
       })
     })
   })
+
+  context('When a one job at a time operative is logged in', () => {
+    beforeEach(() => {
+      cy.loginWithOneJobAtATimeOperativeRole()
+    })
+
+    it('Displays content in the header', () => {
+      cy.visit('/')
+
+      cy.get('.lbh-header__service-name').contains('Repairs Hub')
+      cy.get('.lbh-header__title-link').should('have.attr', 'href', '/')
+
+      cy.contains('Manage work orders').should('not.exist')
+
+      cy.contains('Search').should('not.exist')
+
+      cy.get('#signout')
+        .contains('Sign out')
+        .should('have.attr', 'href', '/logout')
+    })
+
+    context('When they have work orders attached to them', () => {
+      beforeEach(() => {
+        cy.clock(new Date('June 11 2021 13:49:15Z'))
+
+        cy.intercept(
+          {
+            method: 'GET',
+            path: '/api/operatives/hu0001/workorders',
+          },
+          {
+            fixture: 'operatives/workOrders.json',
+          }
+        ).as('operativesWorkOrders')
+
+        cy.intercept(
+          {
+            method: 'GET',
+            path: '/api/workOrders/10000621',
+          },
+          {
+            fixture: 'operatives/workOrder.json',
+          }
+        ).as('operativesWorkOrder')
+
+        cy.intercept(
+          {
+            method: 'GET',
+            path: '/api/properties/00012345',
+          },
+          {
+            fixture: 'properties/property.json',
+          }
+        ).as('property')
+
+        cy.intercept(
+          {
+            method: 'GET',
+            path: '/api/workOrders/10000621/tasks',
+          },
+          {
+            fixture: 'workOrders/task.json',
+          }
+        ).as('task')
+      })
+
+      it('Displays only one work order appointment', () => {
+        cy.visit('/')
+        cy.wait('@operativesWorkOrders')
+
+        cy.get('.lbh-heading-h2').contains('Friday 11 June')
```

Comment:
In a few occasions in this test file "contains" seems to be used to make an assertion, but "contains" itself, to my understanding, is just an easier way to get an item, like "cy.get", using the item's text. It doesn't count as an assertion though.

I would replace all the "contains assertions" with actual assertions using the "should" keyword.

For example:
- cy.contains('Friday 11 June').should('exist')
or
- cy.get('.lbh-heading-h2').should('have.text', 'Friday 11 June')

The test will fail regardless if the text can't be found by the get/contains, so it's not a big deal

Reply:
I assume you would advocate for replacing this across every spec file then, in order to keep the files consistent?

Reply:
If the same "contains assertions" have been used elsewhere yes I would. If you think it's multiple files involved then maybe it's worth adding it as a tech debt item

Reply:
agreed